package.json

```diff
@@ -2,7 +2,7 @@
   "author": "Transcend Inc.",
   "name": "@transcend-io/airgap.js-types",
   "description": "TypeScript types for airgap.js interoperability with custom consent UIs",
-  "version": "10.5.1",
+  "version": "10.6.1",
```

Comment:
nit: either `10.5.2` (since this doesn't really change anything consequential) or `10.6.0`, but not `10.5.1`

Reply:
Yeah, not the first time I made that mistake 🙏🏻